src/WCSimPMTObject.cc

```diff
@@ -220,7 +219,27 @@ G4float PMT20inch::GetmaxQE(){
   return maxQE;
 }
 
+// Should be actual PMT Dark Rate, not effective dark rate in detector including other LE noise
+G4float PMT20inch::GetDarkRate(){
+  /* From e-mail discussion with A.Konaka and S.Nakayama:
+   * SK-I: 4.2 kHz 
+   * SK-IV:5.7 kHz, both before electronics threshold in skdetsim
+   * Measured DN with 0.25 pe threshold:
+   * SK-I: 3.4 kHz  (2003 SK-NIM: 3 kHz. A.Konaka: "2kHz with hot PMTs removed?") 
+   * SK-IV: 4.5 kHz (higher due to FRP)
+   * ToDo: investigate after updating electronics routing, whether to change value to 3.4 kHz
+   */
+
```

Comment:
Ok, so if this is still an open issue perhaps the best thing would be to use the GitHub issue tracker rather than put this in the comments. The issue tracker allows new people to see the discussion and progress on the issue. We also, in general, don't want to be describing our open issues in the comments of the code.